src/org/labkey/test/components/glassLibrary/components/InsertEntitiesPanel.java

```diff
@@ -0,0 +1,193 @@
+package org.labkey.test.components.glassLibrary.components;
```

Comment:
Avoid using the `glassLibrary` package. Use `org.labkey.test.components.labkey.ui`
```suggestion
package org.labkey.test.components.labkey.ui;
```